project/Dependencies.scala

```diff
@@ -36,6 +38,9 @@ object Dependencies {
   object Libraries {
     // Java
     val awsJava          = "software.amazon.awssdk"     %  "s3"                        % V.awsJava
+    val jacksonDatabind  = "com.fasterxml.jackson.core" % "jackson-databind"           % V.jackson // override transitive version to address security vulnerabilities
+    val postgres         = "org.postgresql"             % "postgresql"                 % V.postgresql // override transitive version to address security vulnerabilities
```

Comment:
Doobie is bringing in postgres, but we are already on the latest cats effect 2 version of doobie.